packages/rancher-gatekeeper/generated-changes/patch/values.yaml.patch

```diff
@@ -6,19 +6,19 @@
 +auditInterval: 300
  constraintViolationsLimit: 20
  auditFromCache: false
- disableValidatingWebhook: false
-@@ -8,8 +8,8 @@
+ createNamespace: true
+@@ -11,8 +11,8 @@
  emitAdmissionEvents: false
  emitAuditEvents: false
  image:
 -  repository: openpolicyagent/gatekeeper
--  release: v3.2.1
+-  release: v3.3.0
 +  repository: rancher/openpolicyagent-gatekeeper
-+  tag: v3.2.1
++  tag: v3.3.0
```

Comment:
We need to mirror this image. I will help with that.

Reply:
Here's a PR for that https://github.com/rancher/image-mirror/pull/75